docker-compose.benchmark.yml

```diff
@@ -146,8 +146,8 @@ services:
   test_nakama_benchmark:
     container_name: test_nakama_benchmark
     build:
-      context: ./e2e/testclients/
-      dockerfile: benchmarktestclient/Dockerfile
+      context: ./e2e/tests/
+      dockerfile: bench/Dockerfile
```

Comment:
bench can mean a number of things. It's short and sweet but IMO, unclear. We should use the full word benchmark here to make the meaning crystal clear.

Reply:
i mean, when you run benchmark tests in Go, the command is `go test -bench=.`, so i don't think this would really be that unclear.

Reply:
Good point. You're right. Didn't think of this. If the go command does this then people should know. Approved.